doc/source/intro.rst

```diff
@@ -0,0 +1,136 @@
+Getting started
+---------------
+Gnocchi uses three different back-ends for storing data: one for storing new
+incoming |measures| (the *incoming* driver), one for storing the |time series|
+|aggregates| (the *storage* driver) and one for indexing the data (the *index*
+driver). By default, the *incoming* driver is configured using the value of the
+*storage* driver.
+
+Therefore, before installing Gnocchi, you need to decide which incoming and
+drivers you want to use.
+
+Incoming and storage drivers
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Gnocchi can leverage different storage systems, such as:
+
+* File (default)
+* `Ceph`_ (preferred)
+* `OpenStack Swift`_
+* `Amazon S3`_
+* `Redis`_
+
+Depending on the size of your architecture, using the file driver and storing
+your data on a disk might be enough. If you need to scale the number of server
+with the file driver, you can export and share the data via NFS among all
+Gnocchi processes. In any case, it is obvious that S3, Ceph and Swift drivers
+are largely more scalable. Ceph also offers better consistency, and hence is
```

Comment:
s/ In any case, it is obvious that S3, Ceph and Swift drivers are largely more scalable/Ultimately, the S3, Ceph, and Swift drivers are more scalable storage options.

Reply:
fixed